myaws/ssm_parameter_env.go

```diff
@@ -43,11 +51,12 @@ func formatSSMParameterAsEnv(parameter *ssm.Parameter, prefix string, dockerForm
 	// Drop prefix and get suffix as a key name.
 	suffix := strings.Replace(*parameter.Name, prefix, "", 1)
 	// if first character is period, then drop it.
-	if suffix[0] == '.' {
+	if suffix[0] == '.'|'/' {
```

Comment:
Does this work as intended? `|` is a bitwise OR operator.
Probably you intended like this?
```
if suffix[0] == '.' || suffix[0] == '/' {
```

Comments also should be updated.

Reply:
It seemed to work fine when I tried it. 
But your codes are correct.
I fixed it.